src/Symfony/Component/Intl/Resources/bin/transliterator-emoji-rules.php

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+$sourceFilename = __DIR__.'/emoji-test.txt';
+
+Builder::downloadSource($sourceFilename);
+$rules = Builder::buildRules($sourceFilename);
+Builder::saveRules($rules);
+
+class Builder
+{
+    public static function downloadSource(string $sourceFilename): void
+    {
+        if (!file_exists($sourceFilename)) {
+            copy('https://www.unicode.org/Public/emoji/15.0/emoji-test.txt', $sourceFilename);
```

Comment:
this should download the latest emoji data IMO

Reply:
How to get it? cf: https://www.unicode.org/Public/emoji/

Reply:
I guess it can be done the same way as an update for ICU:
- https://github.com/unicode-org/icu/blob/main/icu4c/source/data/unidata/emoji-sequences.txt
- https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/Intl/Resources/bin/update-data.php

Reply:
you can replace the version number with `latest` to be redirected to the latest version (which is `14.0` currently as Emoji 15.0 is not released yet)

Reply:
I resolved this conversation, see the other comment by @damienalexandre